src/docs/patterns/lists.mdx

```diff
@@ -60,7 +197,142 @@ Occasionally, you may want users to be able to re-order the items in a list manu
+        '3: Global actions in its own button row.',
+    ]}
+/>
+
+Common global list actions include:
+
+-   Adding a new list item
+-   Sorting and filtering the list
+-   Toggling a different view, such as a grid view
+-   Exporting or downloading the list data
+
+### Drag and Drop
+
+<ImageGrid images={[SortableListImage]} regularWidth />
+<DemoCard repository={'sortable-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fsortable-list%2Fsortable-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/sortable-list/index.tsx&initialpath=sortable-list'} ionic reactNative />
+
+Drag and drop is a special case for list sorting. It re-orders the items in a list manually. This should be accomplished by placing a drag handle as the leftmost element in the row. The ability to re-order the list should be locked behind an Edit Mode toggle mechanism.
+
+## Local List Actions
+
+<ImageGrid
+    images={[LocalListActionsImage]}
+    caption={[
+        '1: A more_vert icon, used to open a menu for more list actions.',
+        '2: A chevron, usually opens up a new page with list item details.',
+        '3: A few icons that serve as list action buttons.',
+        '4: List actions can be used together.',
+        '5: An inline checkbox to toggle a status.',
+        '6: An inline text field for data entry.',
+    ]}
+    gridImageProps={{ sm: 8 }}
+/>
+
+If there are actions that you would like to perform on individual list items, you may add interactive elements such as icons or a 3-dot action menu. Depending on the space available, you typically will want to use the action menu if you have more than three actions that can be taken.
+
+> If your list item is clickable for the entire item, then you should not arrange any extra local list actions, as the two interactions can intervene with each other.
+
+### Navigating Down a Tree Structure
+
+<ImageGrid
+    images={[NavigateListImage]}
+    caption={['1: A list used for separated pages.', '2: A list displaying a folder structure.']}
+    regularWidth
+/>
+
+Lists can be used to navigate a tree-like structure in an application. See [the navigation pattern](/patterns/navigation).
+
+### Actions on Selected Items
+
+<ImageGrid images={[MultiSelectImage]} regularWidth />
+<DemoCard repository={'multiselect-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fmultiselect-list%2Fmultiselect-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/multiselect-list/index.tsx&initialpath=multiselect-list'} ionic reactNative />
+
+You may want to take action on several list items at once, but not all. The easiest way to do this is through a multi-select list. You may optionally add a highlight color for the selected items.
```

Comment:
Can we add more content about where to present the actions? The image uses a button panel, but our example uses a snackbar. Do we have recommendations/guidance around this?

Reply:
Sounds like it can be a topic for overlay pattern overhaul :P

All the desktop examples I've seen use a button panel, and all the mobile examples use icon buttons in the app bar.

Reply:
It only goes in overlay if the recommendation is to use overlays 😛

Reply:
Or to not use overlays? 😝